commit 0087047f1fcd30640f487ca3c4c848a304021736
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-08-27T18:08:53-07:00 (6 weeks ago)
    
    joyent/manta-thoth#177 Installer should be more clear about which $SDC_ACCOUNT and $SDC_KEY_ID to use

diff --git a/bin/sdc-thoth-install b/bin/sdc-thoth-install
index 6d5e7de..489fecf 100755
--- a/bin/sdc-thoth-install
+++ b/bin/sdc-thoth-install
@@ -92,8 +92,8 @@ if [[ -z $DOWNLOAD_MANTA ]]; then
 
 fi
 
-[[ -n $SDC_ACCOUNT ]] || fatal "must set \$SDC_ACCOUNT"
-[[ -n $SDC_KEY_ID ]] || fatal "must set \$SDC_KEY_ID"
+[[ -n $SDC_ACCOUNT ]] || fatal "must set \$SDC_ACCOUNT. This is the account in manta that stores your thoth data."
+[[ -n $SDC_KEY_ID ]] || fatal "must set \$SDC_KEY_ID. This must already exist in the $SDC_ACCOUNT account, and needs to be available on this node (ssh-agent is recomended)."
 
 if [[ -z $SDC_URL ]]; then
 	server=us-west-1.api.joyentcloud.com
